lnbits/core/crud.py

```diff
@@ -28,12 +29,45 @@
     User,
     Wallet,
     WebPushSubscription,
+    createUser,
 )
 
 # accounts
 # --------
 
 
+async def create_user(data: createUser) -> User:
+    if await get_account_by_email(data.email):
+        raise Exception("user exists")
+
+    pwd_bytes = data.password.encode("utf-8")
+    salt = gensalt()
+    password = hashpw(pwd_bytes, salt)
+
+    user_id = uuid4().hex
+    await db.execute(
```

Comment:
- should not allow duplicate emails

Reply:
its checked on line, 39 above, what do you mean?